app/src/main/res/values/dimens.xml

```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="spacing_normal">16dp</dimen>
+
+    <!--
+    Refer to App Widget Documentation for margin information
+    https://developer.android.com/guide/topics/appwidgets/index.html#CreatingLayout
+    -->
+    <dimen name="widget_margin">0dp</dimen>
```

Comment:
Where is this used?

Reply:
This constant isn't used. Probably a remnant of support of pre-4.0 Android. I will remove it.